app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,51 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
```

Comment:
the naming convention of the modules should be constant as you keep changing it , it is better to save at as Api::V1::BooksController